hack/make/validate-vet

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source "$(dirname "$BASH_SOURCE")/.validate"
+
+IFS=$'\n'
+files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep -v '^vendor/' || true) )
+unset IFS
+
+for f in "${files[@]}"; do
+	# we use "git show" here to validate that what's committed is vetted
+    NOT_VETTED=$(git show "$VALIDATE_HEAD:$f" | go vet)
+	if [ $NOT_VETTED ]; then
+        FAILS=yes
+        echo $NOT_VETTED
+	fi
```

Comment:
Yeah, I agree with that -- but the format of the entire script output is
inconsistent with our other validate scripts.